applications/app/controllers/QuizController.scala

```diff
@@ -62,7 +62,7 @@ class QuizController(contentApiClient: ContentApiClient)(implicit context: Appli
     val result = contentApiClient.getResponse(capiQuery) map { itemResponse =>
       val maybePage: Option[QuizAnswersPage] = itemResponse.content.flatMap { content =>
 
-        val quiz = Atoms.make(content).flatMap(_.quizzes.find(_.id == quizId))
+        val quiz = Atoms.make(content, maybePageShares = None).flatMap(_.quizzes.find(_.id == quizId))
```

Comment:
I would set a default value to None to the `maybePageShares` field so you don't have to specify it when you don't need them